pkg/ibmcsidriver/controller_helper_test.go

```diff
@@ -40,54 +40,68 @@ func TestGetRequestedCapacity(t *testing.T) {
 	testCases := []struct {
 		testCaseName  string
 		capRange      *csi.CapacityRange
+		profileName   string
 		expectedValue int64
 		expectedError error
 	}{
+		{
+			testCaseName:  "Check minimum size supported by volume provider in case of nil passed as input for sdp profile",
```

Comment:
please add negative tests as well

Reply:
Added few cases covering nil but  we cannot pass negative value to this parameter. IDeally this will come from the k8s parsing form PVC.It is not allowed